tabcmd.spec

```diff
@@ -0,0 +1,44 @@
+# -*- mode: python ; coding: utf-8 -*-
+
+
```

Comment:
Is this spec related to building the python exe?

Reply:
yes - currently it's all just the default settings from pyinstaller.